library/corundum/scripts/sync_reset.tcl

```diff
@@ -0,0 +1,7 @@
+###############################################################################
+## Copyright (C) 2025 Analog Devices, Inc. All rights reserved.
+### SPDX short identifier: ADIBSD
+###############################################################################
+
+set_false_path \
```

Comment:
This constraint-related script can be removed as it isn't used in the modified project

Reply:
Yes, missed that